docs/index.html

```diff
@@ -2,13 +2,13 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8">
-		<title>three.js docs</title>
+		<title>three.js docs/examples</title>
 		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 		<link rel="shortcut icon" href="../files/favicon_white.ico" media="(prefers-color-scheme: dark)"/>
 		<link rel="shortcut icon" href="../files/favicon.ico" media="(prefers-color-scheme: light)" />
 		<link rel="stylesheet" type="text/css" href="../files/main.css">
 		<!-- console sandbox -->
-		<script src="../build/three.min.js" async defer></script>
+		<!-- <script src="../build/three.min.js" async defer></script> -->
```

Comment:
This was here because some users wanted to be able to open the browsers devtools and test things with the API there.

Reply:
Ah I see, like testing what's in the docs. Understood. I will revert it.